docassemble/AssemblyLine/data/questions/ql_baseline.yml

```diff
@@ -1672,6 +1701,11 @@ question: |
   You have ${ x.number_gathered() } document(s) so far. Do you have another document
   you want to upload?
 subquestion: |
+  % if hasattr(x, 'maximum_size'):
+  The total size of all exhibits must be less than ${ humanize.naturalsize(x.maximum_size) }.
+  You can upload ${ humanize.naturalsize(x.maximum_size - x.size_in_bytes())} more.
+  % endif
+
```

Comment:
I don't love this wording, but I don't have any better yet and it's better to warn in advance than do nothing.

Reply:
Definitely agree that it could be better. I had to avoid "All exhibits must be less than ...", because that reads like each exhibit individually needs to be less than that.